docs/stdlib/set.rst

```diff
@@ -503,10 +511,11 @@ Sets
 
     :index: aggregate
 
-    Generalized boolean :eql:op:`and` applied to the set of *values*.
+    Returns a :eql:type:`bool` value with :eql:op:`and` applied
+    to all values.
```

Comment:
```suggestion
    Applies generalized boolean :eql:op:`and` to a set of values.
```

Reply:
```suggestion
    Returns ``true`` if all values in the given set are ``true``.
```

If this works, maybe the following lines should also be rewritten to just talk about the case of the empty set.